classes/01class/exercises/c01-e03/henriqueediniz/c01-e03.txt

```diff
@@ -0,0 +1,59 @@
+Public IP:
+
+I access the EC2 console and with help of documentation I made a configuration of my first instance. I create one Image of Amazon Linux AMI with a default VPC and I create a security group called Henrique Security Group to allows SSH with Key pairs that I create before named as myKey.
+
+My terminal:
+
+✦ 👽 henriquediniz ssh -i "myKey.pem" ec2-user@ec2-34-238-83-178.compute-1.amazonaws.com
+Last login: Fri Feb  7 12:07:15 2020 from 155.143.178.112
+
+       __|  __|_  )
+       _|  (     /   Amazon Linux 2 AMI
+      ___|\___|___|
+
+https://aws.amazon.com/amazon-linux-2/
+7 package(s) needed for security, out of 39 available
+Run "sudo yum update" to apply all updates.
+[ec2-user@ip-172-31-85-181 ~]$ 
+
+
+
+Documentation reference: https://docs.aws.amazon.com/efs/latest/ug/gs-step-one-create-ec2-resources.html
+
+
+Private IP: 
+
+I create a Security Group to my Jump Server and I create a Jump Server instance with my SSH. After, I have uploaded my SSH with Putty to my instance that I would like to connect and tried to connect.
+
+
+
+
+My terminal in putty: 
+
+login as: ec2-user
+Authenticating with public key "myKey.pem" from agent
+Last login: Sun Feb 16 10:32:45 2020 from ec2-18-206-107-26.compute-1.amazonaws.com
+
+       __|  __|_  )
+       _|  (     /   Amazon Linux 2 AMI
+      ___|\___|___|
+
+https://aws.amazon.com/amazon-linux-2/
+[ec2-user@ip-172-31-61-19 ~]$ ssh ec2-54-175-0-167.compute-1.amazonaws.com
+
+       __|  __|_  )
+       _|  (     /   Amazon Linux 2 AMI
+      ___|\___|___|
+
+https://aws.amazon.com/amazon-linux-2/
+No packages needed for security; 2 packages available
+Run "sudo yum update" to apply all updates.
+[ec2-user@ip-172-31-81-119 ~]$
+
+
+
+
+
+I had use this video to show the steps to create a jump server and access my instance: https://www.youtube.com/watch?v=Mwf17O45IA0
```

Comment:
Would you like to add this reference link to the exercise documentation to help other students? If you have some time create a PR to contribute :)